buildroot-external/package/lxd-guest-agent/lxd-guest-agent.mk

```diff
@@ -0,0 +1,21 @@
+################################################################################
+#
+# lxd-guest-agent
+#
+################################################################################
+
+LXD_GUEST_AGENT_DEPENDENCIES = host-pkgconf
+
+define LXD_GUEST_AGENT_INSTALL_INIT_SYSTEMD
+	$(INSTALL) -D -m 644 $(LXD_GUEST_AGENT_PKGDIR)/lxd-agent.service \
+		$(TARGET_DIR)/usr/lib/systemd/system/lxd-agent.service
+	$(INSTALL) -D -m 755 $(LXD_GUEST_AGENT_PKGDIR)/lxd-agent-setup \
+		$(TARGET_DIR)/usr/lib/systemd/lxd-agent-setup
+	$(INSTALL) -d $(TARGET_DIR)/etc/systemd/system/multi-user.target.wants
+	ln -fs /usr/lib/systemd/system/lxd-agent.service \
+		$(TARGET_DIR)/etc/systemd/system/multi-user.target.wants/lxd-agent.service
+        $(INSTALL) -D -m 644 $(LXD_GUEST_AGENT_PKGDIR)/99-lxd-agent.rules \
+                $(TARGET_DIR)/usr/lib/udev/rules.d/99-lxd-agent.rules
```

Comment:
FWIW, there also linter errors on those two lines, see:
https://github.com/home-assistant/operating-system/actions/runs/3079333274/jobs/4979283480

Reply:
Fixed / Removed those rules as I have the systemd patch in place now.